distributions/aws/examples/storage/fsx-for-lustre/static-provisioning/fsx-pv.yaml

```diff
@@ -0,0 +1,19 @@
+apiVersion: v1
```

Comment:
suggest removing the fsx prefix in the file names since its already under fsx directory

Reply:
makes sense.

Reply:
done.